### PR TITLE
Display a 404 on people/:id and company/:id when id does not exist

### PR DESCRIPTION
**Pull request by PepeGonzale**
#1378

### CODE REVIEW

**Review by lucasbordeau — COMMENTED** [review]
Nice work

File: server/.env.example [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Use this for local setup
-PG_DATABASE_URL=postgres://twenty:twenty@localhost:5432/default?connection_limit=1
+PG_DATABASE_URL=postgres://postgres:1234@localhost:5432/twenty?connection_limit=1
```
[the remark this answers is not in the document]
Comment (PepeGonzale) [reply server/.env.example:2]: Yes

[the remark this answers is not in the document]
Comment (PepeGonzale) [reply server/.env.example]: @lucasbordeau I update the .env.example

[the remark this answers is not in the document]
Comment (PepeGonzale) [reply server/.env.example]: @lucasbordeau I update the .env.example

**Review by lucasbordeau — APPROVED** [review]

**Comment by lucasbordeau** [comment]
@PepeGonzale Feel free to come on the discord server to chat with us ! https://discord.gg/rYvMveY2

**Pull request merged**